avogadro/core/molecule.h

```diff
@@ -533,6 +533,12 @@ class AVOGADROCORE_EXPORT Molecule
   int coordinate3d() const;
   bool setCoordinate3d(const Array<Vector3>& coords, int index);
 
+  /**
+   * Timestep property is used when molecular dynamics trajectories are read
+   */
+  bool setTimeStep(double timestep, int index);
+  bool getTimeStep(double& timestep, int index);
```

Comment:
We follow Qt naming convention, setTimeStep, but timeStep for the getter. I would also prefer to follow Qt's convention on the getter - return a double as one would expect, but have an optional bool passed in as the final argument, by reference, that can indicate if the time step was valid.